artifact/image/image.go

```diff
@@ -19,14 +19,16 @@ import (
 )
 
 type Artifact struct {
-	image image.Image
-	cache cache.ArtifactCache
+	image           image.Image
+	cache           cache.ArtifactCache
+	skipDirectories []string
 }
 
-func NewArtifact(img image.Image, c cache.ArtifactCache) artifact.Artifact {
+func NewArtifact(img image.Image, c cache.ArtifactCache, skipDirectories ...string) artifact.Artifact {
```

Comment:
```suggestion
func NewArtifact(img image.Image, c cache.ArtifactCache, skipDirectories []string) artifact.Artifact {
```

This parameter will be passed from a CLI option in Trivy. In that case,  `skipDirectories` should be already []string. I think we don't have to use a variadic function.